pasqal_cloud/__init__.py

```diff
@@ -54,6 +55,38 @@
 from pasqal_cloud.utils.responses import JobCancellationResponse, PaginatedResponse
 from pasqal_cloud.workload import Workload
 
+from ._version import __version__, deprecation_date
+
+DEPRECATION_WARNING_PERIOD = timedelta(days=30)
+
+# Ensure DeprecationWarnings are not filtered out at runtime
+simplefilter("always", DeprecationWarning)
+
+
+def _check_sdk_version() -> None:
+    # Compare current date to deprecation date to check whether
+    # current version has fallen out of maintenance window.
+    current_date = datetime.now()
```

Comment:
I am wondering about the UTC, as the releases were done in UTC+1 how does it work if someone from the US for example, call this method?

Reply:
Added tzinfo